.github/ISSUE_TEMPLATE/use-case-template.yml

```
@@ -0,0 +1,95 @@
+name: Use Case Proposal for Gaps
+description: Use this template to provide a use case to motivate standardization work for a gap in the WoT specifications
+labels: ["UC"]
+title: "Provide a title for your use case..."
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > [!Note]
+        Thank you for proposing a new use case to motivate working on the gaps of WoT specifications! Use case proposals use a structured GitHub issue so that we can semi-automate the process. A bot will check the suggestion after creation, and report on missing properties or other problems before we review the suggestion.
+  - type: input
+    id: submitter
+    attributes:
+      label: Submitter Contact Information
+      description: Please enter your desired form of contact (e.g. Email, GitHub account name). Write "Separate" if you want to provide this in another way.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ## Background
+  - type: input
+    id: domain
+    attributes:
+      label: Domain or Industry
+      description: If your use case is from a specific domain, please provide here.
+    validations:
+      required: false
+  - type: textarea
+    id: introduction
+    attributes:
+      label: Introduction
+      description: Give a basic, non-technical introduction of the use case
+    validations:
+      required: true
+  - type: textarea
+    id: wot-usage
+    attributes:
+      label: WoT Usage
+      description: How do you already use the Web of Things specifications such as Thing Description, Discovery, Bindings, Profiles, Scripting API? If you are not using WoT, please write "None".
```

I suggest linking each of these titles to their web location.
```suggestion
      description: How do you already use Web of Things specifications such as Thing Description, Discovery, Bindings, Profiles, Scripting API? If you are not using WoT, please write "None".
```